test/websocket.ganache.js

```diff
@@ -535,7 +533,6 @@ describe('WebsocketProvider (ganache)', function () {
             web3.currentProvider.once('reconnect', async function () {
                 try {
                     await web3.eth.getBlockNumber()
-                    assert.fail()
```

Comment:
I think it is better practice to keep this line. Because otherwise, if the previous line did not throw, the test will continue to run for 6 seconds before timing out.